frontend/src/components/Checkbox/index.js

```diff
@@ -0,0 +1,29 @@
+import { PropTypes } from "prop-types";
+
+import styles from "./index.module.css";
+
+function Checkbox({ id, children }) {
+  return (
+    <>
+      <input id={id} className={styles.input} type="checkbox"></input>
+      <label htmlFor={id}>{children}</label>
+      <span className={styles.control}>
+        <svg viewBox="0 0 24 24" aria-hidden="true" focusable="false">
+          <path
+            fill="none"
+            stroke="currentColor"
+            strokeWidth="4"
+            d="M1.73 12.91l6.37 6.37L22.79 4.59"
+          />
+        </svg>
```

Comment:
Is this the SVG that I used in de CodeSandbox?

Reply:
Will change it then. Gonna export it from Hannah's design